web/src/stream_data.ts

```diff
@@ -186,6 +187,14 @@ export function get_sub_by_id(stream_id: number): StreamSubscription | undefined
     return stream_info.get(stream_id);
 }
 
+export function maybe_get_creator_details(creator_id: number | null): User | undefined {
+    if (creator_id === null) {
+        return undefined;
+    }
+
+    return people.get_by_user_id(creator_id);
```

Comment:
I think this is an appropriate application for `get_user_by_id_assert_valid` -- we know the user exists, we just may not have access to it.

Reply:
Update to use `get_user_by_id_assert_valid` function.